### ISSUE TITLE
Please Add a smaller button

### ISSUE THREAD

**Issue by Three50freak** [tsunamayo/Starship-EVO#5279]
### Short Description of the issue:

... No issues.  You have an amazing game.  May I ask you to give us a 1/8 button.  That would be awesome.  Thank you for your time and consideration. 

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by EmceeAlpin** [comment]
I second this request!

**Comment by ProPeach** [comment]
I'll always support this kind of request 😎
It really does seem like there's a broad appetite for them, even among newer players, especially considering the ever expanding uses for logic with more survival and gameplay mechanics on the way. We only have so many hotkeys, and cockpit get very cramped quickly with 0.25m buttons/switches.

Speaking of cramped cockpits and smaller bricks, did you manage work out a way of slimming down the Joystick brick Tsuna? I know a while ago in the feedback on patch #5015 you mentioned you would try, but it'd be really cool to see the Joystick fit a bit better in the hand. Womble's render using the existing Yoke grip with a 1/8th base looked excellent.

![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/7168488e-d482-438d-a486-e74e84773a2c)

**Comment by TIKIRobo** [comment]
I agree, i love the 1/4 buttons a lot but also like Peach said it would be so nice to have the smaller sizes for cramped cockpits or on the doors of small spaces, same with switches
